code-studio/scripts/build-commands.js

```diff
@@ -295,3 +337,25 @@ function warnIfWrongNodeVersion() {
       console.log('You are using node ' + process.version + '. This build script expects ' + engines_node + '.');
   }
 }
+
+/**
+ * Wraps a function so that calling it with a set of arguments returns a new
+ * function with those arguments pre-bound - essentially a one-time curry.
+ * @param {!function} fn
+ * @returns {Function}
+ */
+function makeBuildCommand(fn) {
+  return function () {
+    return fn.bind.apply(fn, [undefined].concat([].slice.call(arguments)));
+  };
+}
```

Comment:
No - A function wrapped by _.curry will invoke as soon as it has all the arguments it needs, so in cases were we're providing _all_ arguments we wouldn't get the deferred invocation we need.  Conversely, _.curry doesn't work nicely for functions with optional arguments, as it will keep deferring until all arguments are provided.  We really want this custom "one-time" curry.